src/db/archive-node-adapter/queries.ts

```diff
@@ -91,6 +91,7 @@ function emittedZkAppCommandsCTE(db_client: postgres.Sql) {
   emitted_zkapp_commands AS 
   (
     SELECT blocks_accessed.*,
+            zkcu.id AS zkapp_account_update_id,
```

Comment:
what's zkcu.id?

Reply:
Refers to this field here: https://github.com/MinaProtocol/mina/blob/2f2da075071d5b4b6f64749efd8b70d8776688e3/src/app/archive/zkapp_tables.sql#L261

Reply:
Nice!